resources/meetups/2022-05-19-laravel-poland-meetup-22.json

```diff
@@ -27,7 +27,7 @@
       "speakers": [
         {
           "person": "@patrykfilipiak",
-          "company": null
+          "company": "@freelancer"
```

Comment:
I think I would rather stay with nullable company here. This could implicate that there's a company named Freelancer somewhere.